packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/ios/Podfile

```diff
@@ -30,15 +32,17 @@ flutter_ios_podfile_setup
 target 'Runner' do
   use_frameworks!
   use_modular_headers!
-  pod 'FirebaseFirestore', :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', :tag => '8.9.0'
+  {{#push_notifications}}
+  pod 'FirebaseFirestore', :git => 'https://github.com/invertase/firestore-ios-sdk-frameworks.git', :tag => $FirebaseSDKVersion
```

Comment:
What's the reason of using a non-official FirebaseFirestore ?

Reply:
I've provided one response for this question [here](https://github.com/Prime-Holding/rx_bloc/pull/228#discussion_r918951896).

Regarding the `non-official` part of your question, there is no official repo for this. This repository is maintained by a person unrelated to firebase, google or apple. However the repository was acknowledged by the FlutterFire website. This line of code was a community suggestion for when dealing with firebase in flutter, in order to reduce build times.